tools/configs/golangci.yaml

```diff
@@ -151,6 +152,67 @@ issues:
       linters:
         - goheader
 
+    # Exclude running force type assert check in these paths (should likely fix them all)
+    # Note: The last item must not have a `|` at the end otherwise linter ignores everyfile.
+    - path: "(\
```

Comment:
question: Why would we want this?

Reply:
Is kind of the last PR point "discussion" I wanted to hear opinions about. It's because there are many linter complaints in those files I didn't want to resolve in this PR. These should absolutely be all resolved so this list is empty.

Reply:
Ah my bad, I didn't read the description carefully enough - I'm totally okay with this then, so long as this is documented in the comment, so it is very obvious to anyone reading that they should work towards reducing the list, not expanding it.

Reply:
Agreed will document that in the comment :)

- [ ] Document more clearly we need to reduce the list not expand it.

Reply:
Resolved in commit: `PR(ADHERE): Adhere to code review - better documentation`